site/docs/scale.md

```diff
@@ -186,6 +186,10 @@ For ordinal, quantitative, and time fields, `range` can be a two-element array d
 
 <div class="vl-example" data-name="scatter_color_ordinal_custom"></div>
 
+We can also create diverging color graph by specify `range` with multipe elements.
```

Comment:
1) typo `multipe` => `multiple`

2) Maybe add 
```
### Piecewise Scale Example: Diverging Color Scale
```
3) in the example you also have to specify `domain` to match `range`, but that is not described here. 

Maybe look at how Mike describe about piece wise case there so people understand the constraint better:
https://github.com/d3/d3-scale#continuous_domain